### ISSUE TITLE
SessionHandler::read(): Session data file is not created by your uid

### ISSUE THREAD

**Issue by coreycondardo** [AdventureLookup/AdventureLookup#243]
Upon initial start of the app (after following the readme), using Vagrant I got the following error:
```
SessionHandler::read(): Session data file is not created by your uid
```
I was able to get things up and running by following this stack overflow suggestion:
```
Editing config/config.yml and replacing save_path: "%kernel.root_dir%/../var/sessions/%kernel.environment%" with save_path: /var/lib/php/sessions fixed the error.
```
source: https://stackoverflow.com/questions/39980171/symfony-3-1-5-warning-sessionhandlerread-session-data-file-is-not-created

Is there something I did wrong or is this something another contributer may run into?

**Comment by burnhamrobertp** [comment]
Are you running in vagrant? Are you using `php bin/console server:start 0.0.0.0:8000` to start the dev server?

The error in question only occurs if PHP cannot write out a session data file to disk (usually a permissions error or a path error). I've not seen this error before in using vagrant and symfony's dev server.

**Comment by coreycondardo** [comment]
Yes, I am doing both of those things. I followed the README exactly.

Running:
OSX 10.12.6
Vagrant 2.0.1
Virtualbox: 5.2.0 r118431

I believe everything else would be contained within the vagrant box itself. I got it working so no big deal, but it might be worth keeping an eye on if someone else has an issue. Smooth on boarding is always nice :). I will say, the readme and the way this is all setup generally was super smooth, so thanks for that!
